.travis.yml

```diff
@@ -37,14 +37,17 @@ matrix:
       - PRODUCT_TESTS='-g hdfs_impersonation'
       - CLUSTER_TYPE='singlenode-hdfs-impersonation'
     - env:
-      - PRODUCT_TESTS='-g hdfs_impersonation,authorization'
+      - PRODUCT_TESTS='-g hdfs_impersonation,authorization,storage_formats,cli'
```

Comment:
Yes. `storage_formats` verifies that every single Hive format supported by Presto works fine with kerberos.